Sources/SwiftProtobuf/Map.swift

```diff
@@ -17,7 +17,7 @@
 import Foundation
 import Swift
 
-public struct ProtobufMap<KeyType: MapKeyType, ValueType: MapValueType>
+public struct Map<KeyType: MapKeyType, ValueType: MapValueType>
```

Comment:
In general I'm supportive of simplifying these names (I'm still iffy about `Message`, but we can see if it causes problems for people downstream), but this one feels wrong to me. Technically this type is not a "map", but more like a type descriptor of sorts. I feel like we need to capture that in some way in the name (especially since the analogous types, like `ProtobufInt32`, will need some kind of suffix on their names to work if you want to remove `Protobuf` there).

`Descriptor` would be the wrong word to use here though, since that's a loaded term in protobuf. `MapField`/`Int32Field`/etc?